websites/S/Stoodi/presence.ts

```diff
@@ -0,0 +1,131 @@
+const presence = new Presence({
+		clientId: "1043638037042712637",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	strings = presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+	}),
+	images = {
+		play: "https://i.imgur.com/o4FaGfB.png",
+		pause: "https://i.imgur.com/kK39yND.png",
+	};
```

Comment:
Prefer using `enum` for these instead.
Something like this:
https://github.com/PreMiD/Presences/blob/4f297578140efdf9ec96225629c892008df3839e/websites/C/Crunchyroll/presence.ts#L52-L58

Reply:
ohhh, okay